batchlib/workflows/cell_analysis.py

```diff
@@ -200,12 +201,20 @@ def run_cell_analysis(config):
                 'serum_key': serum_key,
                 'marker_key': marker_ana_in_key,
                 'cell_seg_key': config.seg_key,
-                # 'outlier_predicate': outlier_predicate,
                 'write_summary_images': True,
                 'scale_factors': config.scale_factors,
                 'identifier': identifier},
             'run': {'force_recompute': False}}))
 
+    # TODO
+    # - we need to filter out the mean-with-nuclei and sum-without-nuclei results
+    # - choose the correct reference table !
+    job_list.append((MergeAnalysisTables, {
+        'build': {'input_table_names': table_identifiers,
+                  'reference_table_name': table_identifiers[0]}
+    }))
+
+    # TODO figure out what this should do now that we merge the tables
```

Comment:
Merging the tables won't break the DB job, but would make the results unnecessary nested in in the DB (DbResultWriter assumes multiple result files). We just have to bring the code back to it's original implementation (single `input_file`). It suffices to replace those lines https://github.com/hci-unihd/batchlib/blob/master/batchlib/mongo/result_writer.py#L100-L109 with:
```
assert len(input_files) == 1, f"Expected a single table hdf5 file, but {len(input_files)} were given"
input_file = input_files[0]
result_tables = _get_analysis_tables(input_file)
```

Reply:
Looks good, I will do it later.

Reply:
Done.